--- "C:/Users/alex4/AppData/Roaming/Blender Foundation/Blender/5.1/extensions/user_default/io_scene_kotor/scene/armature.py.bak"	2026-06-30 11:09:19.149788000 -0500
+++ "C:/Users/alex4/AppData/Roaming/Blender Foundation/Blender/5.1/extensions/user_default/io_scene_kotor/scene/armature.py"	2026-06-30 11:10:01.525578300 -0500
@@ -105,7 +105,7 @@
         if not anim_data.action_slot:
             anim_data.action_slot = action_slot
 
-    action.fcurves.clear()
+    AnimationNode.clear_fcurves(action)
     apply_object_keyframes_to_armature(mdl_root, armature, action)
     bpy.ops.object.mode_set(mode="OBJECT")
 
@@ -214,7 +214,7 @@
             if not anim_data.action_slot:
                 anim_data.action_slot = action_slot
 
-        action.fcurves.clear()
+        AnimationNode.clear_fcurves(action)
 
         assert bpy.context.scene.frame_current == 0
         rest_location = obj.location.copy()
